terraform/environments/analytical-platform-compute/iam-policies.tf

```
@@ -174,3 +174,86 @@ module "gha_mojas_airflow_iam_policy" {
 
   policy = data.aws_iam_policy_document.gha_mojas_airflow.json
 }
+
+data "aws_iam_policy_document" "analytical_platform_share_policy" {
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "lakeformation:GrantPermissions",
+      "lakeformation:RevokePermissions",
+      "lakeformation:BatchGrantPermissions",
+      "lakeformation:BatchRevokePermissions",
+      "lakeformation:RegisterResource",
+      "lakeformation:DeregisterResource",
+      "lakeformation:ListPermissions",
+      "lakeformation:DescribeResource",
+
```

Pretty sure we will need this
```suggestion
      "lakeformation:GetDataAccess",
```